### PR TITLE
🚢 🐳  Dockerization

### PR DESCRIPTION
**Pull request by g--**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='gherkin-diff',
+    version='0.0.1',
+    author="Ian",
```
Comment (ianmarcinkowski) [inline setup.py]: Ian Marcinkowski ;)

**Comment by minichate** [comment]
:+1:

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM quay.io/freshbooks/python
+RUN apt-get update && apt-get install -y \
+		build-essential \
```
Comment (ianmarcinkowski) [inline Dockerfile]: Do we need `build-essential`?  It's huge

**Comment by ianmarcinkowski** [comment]
:+1:

**Comment by ianmarcinkowski** [comment]
:ship:

**Pull request merged**